TypoSearch.py

```diff
@@ -115,29 +112,30 @@ def main():
                         required=True)
     parser.add_argument("-table", action="store", dest="table", help="Name of table to search in", required=True)
     parser.add_argument("-rank", action="store", dest="rank", help="Rank level to start searching at", required=True)
+    parser.add_argument("-typo", action = "store", dest = "typo",
+                        help= "Maximum number of character DIFFERENCES between two names to be considered a typo, "
+                              "EX: 'foo' and 'foos!' have a character difference of 2", required= True)
     parser.add_argument("--show", action="store_true", dest="show", help="Print Locality ID's to be deleted to screen")
```

Comment:
is this specific to localities only, or does show also apply to taxa?

Reply:
This applies to taxa as well, the set `-typo_limit` is what the typo limit will be for whatever table is passed in with `-table`.

Reply:
My comment ended up on the wrong line - I was asking about `show`. The help text says "Print Locality ID's..." but does `show` work with other tables as well?

Reply:
Yes, `-show` also works with the table passed in with `-table` (Also this script doesn't delete anything). This mistake has been updated in the help message.